Dockerfile

```
@@ -0,0 +1,45 @@
+ENV PATH="/home/${USERNAME}/.local/bin:/home/${USERNAME}/.pulumi/bin:${PATH}"
+ENV HOME=/home/${USERNAME}
+WORKDIR /home/${USERNAME}
+
+# Switch to the user with specified UID and GID
+USER ${UID}:${GID}
+
+# Install Pulumi
+RUN curl -fsSL https://get.pulumi.com | bash
+
+# Install Poetry
+RUN curl -sSL https://install.python-poetry.org | python3 - && \
```

Hadolint

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)